.github/workflows/stale-message.yml

```diff
@@ -0,0 +1,29 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'Old issues will be closed after 105 days of inactivity. This issue has been quiet for 90 days and is being marked as stale. Reply here or add the label "not-stale" to keep this issue open.'
+          close-issue-message: "This issue is being closed due to inactivity. Is this a mistake? Please re-open this issue or create a new one."
+          stale-pr-message: 'Old PRs will be closed after 30 days of inactivity. This PR has been quiet for 14 days and is being marked as stale. Reply here or add the label "not-stale" to keep this PR open.'
```

Comment:
Same comment on asking users to add a label.

Reply:
The stale bot automatically handles adding and removing the labels, all the user needs to do is update the PR/issue.